core/src/main/java/io/lionweb/lioncore/java/serialization/SerializedJsonComparisonUtils.java

```diff
@@ -12,11 +9,11 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-class SerializedJsonComparisonUtils {
+public class SerializedJsonComparisonUtils {
 
   private SerializedJsonComparisonUtils() {}
 
-  static void assertEquivalentLionWebJson(JsonObject expected, JsonObject actual) {
+  public static void assertEquivalentLionWebJson(JsonObject expected, JsonObject actual) {
```

Comment:
What about making this _not_ static? Then we could override `assertEquals()` and `fail()` in subclasses if we wanted to handle them differently. (These two methods would need to be `protected`, of course.)

Reply:
For the time being, I do not see a situation where we may want to subclass this class, so for simplicity I would keep those methods as static until there is a specific use case